apps/consulting/posts/scaling-python.md

```diff
@@ -0,0 +1,214 @@
+---
+title: 'Scaling Python'
```

Comment:
The talk was originally titled `Scaling Python: Banking Edition`. Does that Work? Otherwise I am fine going with what you have above

Reply:
&lt;nod&gt;, I noted that origin of the post from the intro, but I don't think it works very well as a blog title. It was likely fine for the talk, because it was accompanied by an abstract.

Here, I think it's better to be more specific in the title about what was scaled, and with what tools. That it was a banking application seems less directly relevant to an arbitrary reader scanning down the list of blogs. "Oh, I'm not in banking, so this post doesn't apply to me."

@Katebrack, thoughts on this?

Reply:
I prefer the original title actually @bskinn. Sales is planning to use this post as outreach to potential banking clients so it speaks to that. Like you said, "Scaling Python" is generic so the other half succinctly addresses that.

Reply:
Aha, I didn't have that perspective. Glad I asked!

Reply:
To be clear, @Katebrack -- by "original title" you mean the original title of the talk, "Scaling Python: Banking Edition"?

Reply:
Correct @bskinn

Reply:
Ok, I've updated the title to be "Scaling Python: Banking Edition"